widgetsnbextension/src/widget_output.js

```diff
@@ -66,7 +66,7 @@ var OutputView = widgets.DOMWidgetView.extend({
     },
 
     render: function(){
-        let renderOutput = (outputArea) => {
+        var renderOutput = function(outputArea) {
             this.output_area = new outputArea.OutputArea({
```

Comment:
Thanks, your changes look great! I'll try to test and merge tomorrow. Thanks for finding and correcting this oversight!